pysmt/operators.py

```diff
@@ -55,7 +55,11 @@
 BV_ASHR,                                    # Arithmetic shift right (47)
 ARRAY_SELECT,                               # Array Select (48)
 ARRAY_STORE,                                # Array Store (49)
-ARRAY_VALUE                                 # Array Value (50)
+ARRAY_VALUE,                                # Array Value (50)
+
+DIV,                                        # Arithmetic Division (51)
```

Comment:
NOTE

on the website http://rise4fun.com/z3/tutorialcontent/guide you can read the following:

" Z3 also has support for division, integer division, modulo and remainder operators. Internally, they are all mapped to multiplication. "